MIP/mip-74/README.md

```diff
@@ -0,0 +1,218 @@
+
+!!! warning These values are considered constant in the sequel. There may be updated if needed or if new funds are added to the pools.
+
+The Insurance Fund indirectly rate-limits the transfers i.e., for a given transfer from source chain to target chain the Insurance Fund on the _target chain_ is responsible for the rate limit, and thus we will refer to the `insurance_fund_target`.
+For a transfer from L1 (L2) to L2 (L1) the `insurance_fund_target = insurance_fund_L2` (`insurance_fund_L1`) is responsible for the rate limit.
+
+#### Rate limit on the target chain
+
+The rate limit is dependent on the fund size in the Insurance Fund. In particular the maximum rate limit is defined by
+
+`max_rate_limit_target = insurance_fund_target / reaction_time`,
+
+where the `reaction_time` is the time it takes for the Governance Operator to react to a faulty or compromised component. The `reaction_time` is a parameter that is set by the Governance Operator. 
+
+_Implementation recommendation #1_: The default value is 24h. In the initial implementation this value is fixed to avoid complications in gas.
+
+_Implementation recommendation #2_: The target Native Bridge contract checks at every transfer first, whether the relevant Insurance Fund size has changed before calculating the current rate limit and whether the budget is exceeded.
+
+**(Optional) Direct adjustment of rate limit by Governance Operator**
+The rate limit MAY also be adjusted by the Governance Operator directly by a parameter `rate_reduction_target`. However the Rate Limiter MUST NOT set the rate limit higher than the `max_rate_limit_target`. In equation
+
+`rate_limit_target = rate_reduction_target * max_rate_limit_target`,
+
+where `rate_reduction_target` $\in$ `[0,1]` is a parameter that is set by the Governance Operator. As mentioned the `rate_limit_target` MUST not be larger than `max_rate_limit_target`.
+
+**Summary of Adjustment mechanisms**
+The following are possible ways to adjust the rate limit:
+
+1. The Governance Operator indirectly adjusts the rate limit by adding or removing funds from the Insurance Fund.
+1. The Governance Operator indirectly MAY adjust the rate limit by changing the `reaction_time`. 
+1. The Governance Operator MAY adjust the rate limit by changing the `rate_reduction_target`.
+
+#### Rate limit on the source chain
+
+On the source chain the rate limit MUST be limited by the Governance Operator to match the rate limit on the target chain. if only the target chain would be rate limited users could successfully continue to request transfers on the source chain while the budget on the target chain is already consumed. Consequently the  Relayer would not be capable to complete the transfers.
+
+On the source chain the rate limit MAY be lowered by the Relayer. This is to ensure that the rate limit on the target chain is not exceeded. It also permits the Relayer to catch up in case of the Relayer has been down for some time.
```

Comment:
not exactly an attack vector, but if the relayer is compromised, lowering the rate limit budget could be reverted by the malicious actor and extra steps would be needed to secure the bridge.

Reply:
I think this security concern is covered by the Governance Operator, see the para above.